### DISCUSSION TITLE
The Css units x y coordinate don't match the pdf-lib units

### DISCUSSION THREAD

**Discussion by nafism05** [Hopding/pdf-lib#1629] [Q&A]
I'm still working on a web app project to add signatures to PDFs
for the backend I use pdf-lib
for the frontend I use pdf.js by mozilla to display the pdf, and interact.js to move the signature image

I have a problem with the pdf-lib coordinates in placing the signature image in the pdf

I have read several issues and already know that x y in pdf-lib uses 1\72 inch units
then I tried using 1/72 inch units in the CSS in interact.js
but when I use it in pdf-lib, the units in the css don't match the units in pdf-lib

any ideas?

**Comment by Arun-kumar002** [comment]
@nafism05  did you find any solution for this issue.?

**Reply by MarcRam** [reply]
@nafism05 So, Not a solution but an Observation.
I believe CSS x:0 y:0 in the viewport references the top Left, while PDF documentents reference the bottom left as x:0 y:0 
in points

If your signature is ending up closer to the top of your document, this coud be the cause...
you could probably correct for this by subtracting the Y coordinate from the height of the pdf to have the position from the bottom left

I Would love to hear if this was the issue
